indexer/cmd/indexer-refresh/main.go

```diff
@@ -20,34 +14,8 @@ var (
 )
 
 func main() {
-	// Set up logger with a default INFO level in case we fail to parse flags.
-	// Otherwise the final crtiical log won't show what the parsing error was.
-	log.Root().SetHandler(
-		log.LvlFilterHandler(
-			log.LvlInfo,
-			log.StreamHandler(os.Stdout, log.TerminalFormat(true)),
-		),
-	)
-
-	// TODO https://linear.app/optimism/issue/DX-55/api-implement-rest-api-with-mocked-data
-	// don't hardcode this
-	conf, err := config.LoadConfig("../../indexer.toml")
-
-	if err != nil {
-		log.Crit("Failed to load config", "message", err)
-	}
-
-	log.Debug("Loaded config", "config", conf)
-
-	app := cli.NewApp()
-	app.Flags = flags.Flags
-	app.Version = fmt.Sprintf("%s-%s", GitVersion, params.VersionWithCommit(GitCommit, GitDate))
-	app.Name = "indexer"
-	app.Usage = "Indexer Service"
-	app.Description = "Service for indexing deposits and withdrawals " +
-		"by account on L1 and L2"
+	app := cli.NewCli(GitVersion, GitCommit, GitDate)
```

Comment:
```suggestion
	app := cli.NewCli(GitVersion, GitCommit, GitDate)
	app.Run()
```
Oops forgot to call Run()

Reply:
nvm I called it directly below this comment